sms-authenticator/pom.xml

```diff
@@ -18,6 +18,11 @@
 			<artifactId>keycloak-quarkus-server-app</artifactId>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.googlecode.libphonenumber</groupId>
+			<artifactId>libphonenumber</artifactId>
+			<version>8.13.40</version>
```

Comment:
We should probably activate Github Dependabot to automatically bump versions and receive security updates.